etc/image-input.conf

```diff
@@ -0,0 +1,86 @@
+# Image key bindings
+#
+# Location of user-defined bindings: ~/.config/mpv/image-input.conf
+#
+# Lines starting with # are comments. Use SHARP to assign the # key.
+# Copy this file and uncomment and edit the bindings you want to change.
+#
+# List of commands and further details: DOCS/man/input.rst
+# List of special keys: --input-keylist
+# Keybindings testing mode: mpv --input-test --force-window --idle
+#
+# Use 'ignore' to unbind a key fully (e.g. 'ctrl+a ignore').
+#
+# Strings need to be quoted and escaped:
+#   KEY show-text "This is a single backslash: \\ and a quote: \" !"
+#
+# You can use modifier-key combinations like Shift+Left or Ctrl+Alt+x with
+# the modifiers Shift, Ctrl, Alt and Meta (may not work on the terminal).
+#
+# The default keybindings are hardcoded into the mpv binary.
+# You can disable them completely with: --no-input-default-bindings
+
+#LEFT        script-binding/positioning pan-x -0.2 # pan left
+#DOWN        script-binding/positioning pan-y  0.2 # pan down
+#UP          script-binding/positioning pan-y -0.2 # pan up
+#RIGHT       script-binding/positioning pan-x  0.2 # pan right
+#h           script-binding/positioning pan-x -0.2 # pan left
+#j           script-binding/positioning pan-y  0.2 # pan down
+#k           script-binding/positioning pan-y -0.2 # pan up
+#l           script-binding/positioning pan-x  0.2 # pan right
+#Shift+LEFT  script-binding/positioning pan-x -0.02 # pan left slowly
+#Shift+DOWN  script-binding/positioning pan-y  0.02 # pan down slowly
+#Shift+UP    script-binding/positioning pan-y -0.02 # pan up slowly
+#Shift+RIGHT script-binding/positioning pan-x  0.02 # pan right slowly
+#H           script-binding/positioning pan-x -0.02 # pan left slowly
+#J           script-binding/positioning pan-y  0.02 # pan down slowly
+#K           script-binding/positioning pan-y -0.02 # pan up slowly
+#L           script-binding/positioning pan-x  0.02 # pan right slowly
```

Comment:
0.1 and 0.01 seems better imho.

Reply:
It is impossible to predict the majority's preferences here. I will set what you prefer, unless we can get more feedback.

Reply:
It is just consistent with other keybindings.